coalib/bears/Bear.py

```diff
@@ -261,6 +262,29 @@ def get_metadata(cls):
             omit={"self", "dependency_results"})
 
     @classmethod
+    def __json__(cls):
+        """
+        Override JSON export of ``Bear`` object.
+        """
+        _dict = get_public_members(cls)
+        metadata = cls.get_metadata()
+        non_optional_params = metadata.non_optional_params
+        optional_params = metadata.optional_params
+        _dict["metadata"] = {
+            "desc": metadata.desc,
+            "non_optional_params": [({param: non_optional_params[param][0]}
+                                     for param in non_optional_params)],
+            "optional_params": [({param: optional_params[param][0]}
+                                 for param in optional_params)]}
+
+        # Delete attributes that cannot be serialized
+        unserializable_attributes = ["new_result", "printer"]
+        for attribute in unserializable_attributes:
+            if attribute in _dict:
```

Comment:
Coverage drops as the branch will always be true in the current state. Should I remove this check and directly delete the element from the dict?